newsroom/agenda/views.py

```diff
@@ -91,6 +91,13 @@ def search():
     return send_response('agenda', response)
 
 
+@blueprint.route('/agenda/count')
```

Comment:
it should be probably `agenda_featured/count` and not just `agenda` that's a bit confusing